salt/modules/lxc.py

```diff
@@ -1037,7 +1097,9 @@ def select(key, default=None):
     does_exist = exists(name)
     to_reboot = False
     remove_seed_marker = False
-    if clone_from:
+    if does_exist:
+        pass
```

Comment:
not at all, all if i remove the line, i will have to modify the two underlying elif branches to add an 'and not does_exists' which then results in the same cycles...